blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupHandlerWithESIntegrationTest.java

```diff
@@ -135,7 +135,7 @@ public void testUnknownUnit() throws Exception {
                 baseMillis + 86400000,
                 1600);
         Assert.assertEquals(data.getData().getPoints().size(), 0);
-        Assert.assertEquals(data.getUnit(), "unknown");
+        Assert.assertNull(data.getUnit());
```

Comment:
This may be hard to do, but I'd rather have the getUnit does return something like 'fakeunits', because null may be hiding an exception.